.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules/
 .vscode/
 gcs_service_account_key.json
 .DS_Store
+src/.DS_Store
```

Comment:
You can just put `.DS_Store`, (i.e. what you did on line 8) that will ignore that file in any directory.